src/main/java/Parser.java

```diff
@@ -0,0 +1,121 @@
+import java.util.LinkedList;
+
+public class Parser {
+    
+    public static LinkedList<String> parseAndExecute(String input, TaskList tasks, FileHandler fileHandler) 
+            throws DukeException {
+        //Split input incase it has a command and argument
+        String[] inputArray = input.split(" ", 2);
+
+        String[] cmdArgs = new String[0];
+        Task newTask;
+
+        LinkedList<String> result = new LinkedList<>();
+
+        switch (inputArray[0]) { 
+        case "list":

```

Comment:
Would it be better to have a Command instance to take care of this instead? Having a Parser to add tasks into the task list would not be favourable, given that a Parser should only parse the text.